rules/framework.bzl

```diff
@@ -625,7 +625,7 @@ def _attrs_for_split_slice(attrs_by_split_slices, split_slice_key):
     else:
         return attrs_by_split_slices[split_slice_key]
 
-def _bundle_dynamic_framework(ctx, is_extension_safe, avoid_deps):
+def _bundle_dynamic_framework(ctx, is_extension_safe, avoid_deps, product_type):
```

Comment:
In this function ` _bundle_dynamic_framework` - what is the use case to make it accept a `product` type that's not for a dynamic framework? The code is specific to packing a dynamic one

Reply:
Oh we could just set it directly internally without exposing as a param in this function indeed. I can update that.

Reply:
https://github.com/bazel-ios/rules_ios/pull/741/commits/c209ea96a3ccd838c37a94cfc42a77345bb7a20f